### DISCUSSION TITLE
Is there a way to control the pan and tilt features of the camera through homeassistant?

### DISCUSSION THREAD

**Discussion by geoemm7** [fuatakgun/eufy_security#469] [Q&A]
I have a Eufy Security Indoor Cam 2K and I was wondering if its possible to control the pan and tilt features through the homeassistant. If it is possible can anyone guide me? Thanks in advance.

**Comment by fuatakgun** [comment]
there are PTZ services

**Comment by billnis** [comment]
The entities you are looking for on the E220 are disabled that is the only one of the indoor 2K cameras I have, the other ones are probably the same. you need to look at the Diagnostic section and click on disabled entities and all of the disabled entities will be shown. Click on the one you want to use like PTZ Up and click on the gear at the top, click on the red enable button and then close the entity. After 30 seconds it will be available to use. Repeat this for the PTZ directions and then you can assign them to buttons on your dashboard.
